eclass/gstreamer-meson.eclass

```diff
@@ -77,6 +77,12 @@ gstreamer_get_plugins() {
 		"/^# Feature options for plugins (with|that need) external deps$/,/^#.*$/s;^option\('([^']*)'.*;\1;p" \
 		"${S}/meson_options.txt" || die "Failed to extract options for plugins with external deps"
 	)
+
+	# opencv in gst-plugins-bad is split to gst-plugins-opencv
+	if grep -q "option('opencv'" "${EMESON_SOURCE}"/meson_options.txt ; then
+		GST_PLUGINS_EXT_DEPS="${GST_PLUGINS_EXT_DEPS}
+opencv"
```

Comment:
I think `hls` plugin needs similar treatment, it is not picked by `GST_PLUGINS_*_DEPS`